pkgs/misc/uboot/default.nix

```diff
@@ -136,6 +136,14 @@ in {
     dontStrip = false;
     extraMeta.platforms = lib.platforms.linux;
     extraMakeFlags = [ "HOST_TOOLS_ALL=y" "CROSS_BUILD_TOOLS=1" "NO_SDL=1" "tools" ];
+
+    outputs = [ "out" "man" ];
+
+    postInstall = ''
+      mkdir -p $man/share/man/man1
```

Comment:
How about using `installManPage doc/*.1`? From setup hook `installShellFiles`.

Reply:
I didn't know about this. Thanks for the suggestion; I have applied it.